packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -947,6 +944,33 @@ export function TransferPanel() {
     }
   }
 
+  const trackTransferButtonClick = useCallback(() => {
+    trackEvent('Transfer Button Clicked', {
+      type: isTeleportMode
+        ? 'Teleport'
+        : isDepositMode
+        ? 'Deposit'
+        : 'Withdrawal',
+      isCctpTransfer,
+      tokenSymbol: selectedToken?.symbol,
+      assetType: selectedToken ? 'ERC-20' : 'ETH',
+      accountType: isSmartContractWallet ? 'Smart Contract' : 'EOA',
+      network: childChain.name,
+      amount: Number(amount),
+      amount2: isBatchTransfer ? Number(amount2) : undefined
```

Comment:
i think in all events we would want to identify if the transfer is to custom destination address or to the same address

and as discussed before, we would want to send the selected token's parent chain address along with the symbol to ensure no counterfeit token data are mixed in